docs/contributing/code_of_conduct.rst

```diff
@@ -0,0 +1,232 @@
+.. include:: ../../CODE_OF_CONDUCT.rst
+
+Reporting Guidelines
+====================
+
+If you believe someone is violating the code of conduct we ask that you
+report it to the Learning Equality by emailing
+codeofconduct@learningequality.org. All reports will be kept confidential. In
+some cases we may determine that a public statement will need to be
+made. If that's the case, the identities of all victims and reporters
+will remain confidential unless those individuals instruct us otherwise.
+
+If you believe anyone is in physical danger, please notify appropriate
+law enforcement first. If you are unsure what law enforcement agency is
+appropriate, please include this in your report and we will attempt to
+notify them.
+
+If you are unsure whether the incident is a violation, or whether the
+space where it happened is covered by this Code of Conduct, we encourage
+you to still report it. We would much rather have a few extra reports
+where we decide to take no action, rather than miss a report of an
+actual violation. We do not look negatively on you if we find the
+incident is not a violation. And knowing about incidents that are not
+violations, or happen outside our spaces, can also help us to improve
+the Code of Conduct or the processes surrounding it.
+
+In your report please include:
+
+#. Your contact info (so we can get in touch with you if we need to
+   follow up)
+#. Names (real, nicknames, or pseudonyms) of any individuals involved.
+   If there were other witnesses besides you, please try to include them as
+   well.
+#. When and where the incident occurred. Please be as specific as possible.
+#. Your account of what occurred. If there is a publicly available
+   record (e.g. a mailing list archive or a public Slack logger) please
+   include a link.
+#. Any extra context you believe existed for the incident.
+#. If you believe this incident is ongoing.
+#. Any other information you believe we should have.
+
+What happens after you file a report?
+-------------------------------------
+
+You will receive an email from the Code of Conduct committee
+acknowledging receipt immediately. We promise to acknowledge receipt
+within 24 hours (and will aim for much quicker than that).
```

Comment:
The original wording to me sounds as if there are two receipts, one automatic (immediately), the other "personal" from the committee (within 24 hours). 

If that's not case, let's just drop the first sentence and go with the second.